docs/examples/security/jwt/using_jwt_auth.py

```diff
@@ -22,22 +22,26 @@ class User(BaseModel):
 MOCK_DB: Dict[str, User] = {}
 
 
-# JWTAuth requires a retrieve handler callable that receives the JWT token model and the ASGI connection
-# and returns the 'User' instance correlating to it.
+# JWTAuth requires a retrieve handler callable that receives the JWT token
+# model and the ASGI connection and returns the `User` instance correlating to
+# it.
 #
 # Notes:
-# - 'User' can be any arbitrary value you decide upon.
+# - `User` can be any arbitrary value you decide upon.
```

Comment:
The quotes here are intentional as well. Since this isn't rendered anywhere, backticks don't have a special representation, so using standard quotes is better for readability

Reply:
Well, the term "readability" sometimes can be quite subjective.
As I see it, quotes and double quotes are used for grammatical constructs, so words or phrases wrapped in backticks (be they single or double) indicates that the value is related to code (in this case it is the name of a class). It would be less debatable if instead of just 'User' it would be "The class name 'User' [..]" (or "Name of the class 'User' [..]"; I am not too sure which would be more correct).

But anyways, your project - your rules.